src/WorldClock.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+import "./App.css";
```

Comment:
no need to import here, since it is already imported in App. App contains WorldClock so it will definitely have this css file loaded.